kratos/elements/levelset_convection_element_simplex_algebraic_stabilization.h

```diff
@@ -0,0 +1,434 @@
+//    |  /           |
```

Comment:
Why not header and source file separation?

Reply:
agree, this is a must for the core!